kinova_driver/src/kinova_api.cpp

```diff
@@ -329,9 +329,7 @@ int KinovaAPI::initializeKinovaAPIFunctions(KinovaAPIType connection_type)
 
     setTorqueInactivityType = (int (*)(int))initCommandLayerFunction("SetTorqueInactivityType");
 
-    setJointZero = (int (*)(int))initCommandLayerFunction("SetJointZero");
-
-        // %EndTag(experimental)%
+    //setJointZero = (int (*)(int))initCommandLayerFunction("SetJointZero");
```

Comment:
Dead code?

Reply:
It's how Kinova manage functions implementation. They leave functions pointers, and the one that doesn't have an implementation commented.

Reply:
it doesn't really help since git keeps the history. People who read the code behind will read commented code without knowing why it is there.

Reply:
I can leave a more descriptive comment to indicate why it's commented

Reply:
Not a fan of adding a comment for a signe line of unused code. When/if it will be implemented, the programmer won't forget to use it.

Reply:
I think we can leave this as is, there are comments above the multiple api functions specifying when and when not to use these functions